src/Core/Grid/Query/ProfileQueryBuilder.php

```diff
@@ -31,7 +31,7 @@
 use PrestaShop\PrestaShop\Core\Grid\Search\SearchCriteriaInterface;
 
 /**
- * Class ProfilesQueryBuilder is responsible for building queries for profiles grid data.
+ * Class ProfileQueryBuilder is responsible for building queries for profiles grid data.
```

Comment:
```suggestion
 * Responsible for building queries for profiles grid data.
```
Not mandatory ofc, but I find it useless to repeat class name. (furthermore, if you rename the class, there is a high chance that comment will get outdated and misleading)